### ISSUE TITLE
releases?

### ISSUE THREAD

**Issue by bigdimboom** [kzampog/cilantro#26]
Will there be a release version. How long would author keep maintain this library?

**Comment by kzampog** [comment]
Hi!

There are some things I would like to add/change before considering a 'release' version. This is a side project I work on in my free time. I still enjoy it a lot! :)

**Comment by wannesvanloock** [comment]
Lately there have been some breaking changes that make upgrading depending code somewhat more cumbersome. It might be a good time to have some beta releases and a clear changelog as more people are starting to depend on cilantro.

**Comment by allenh1** [comment]
> It might be a good time to have some beta releases and a clear changelog as more people are starting to depend on cilantro.

I agree completely -- it would be great to have a version number to refer to!

**Comment by kzampog** [comment]
Thanks for the reminder, and apologies for this taking so long; this is still in my TODO list. There are a few half-baked changes I would like to wrap up (shouldn't take long), after which we'll have an initial version :D

**Comment by wannesvanloock** [comment]
Would be great! If there's anything we can help with, let us know.

**Comment by allenh1** [comment]
> There are a few half-baked changes I would like to wrap up (shouldn't take long), after which we'll have an initial version :D

Great! I can also contribute wherever you need!
